app/src/main/res/values/strings.xml

```diff
@@ -1138,4 +1138,7 @@ If you are overwhelmed by the number of quests, you can always fine-tune which q
 Alternatively, you can leave a note (with a photo)."</string>
     <string name="quest_shoulder_title">"Does this road have a shoulder?"</string>
     <string name="quest_fireHydrant_diameter_ref_title">What diameter is specified on the sign for this fire hydrant? (reference number: %s)</string>
+    <string name="link_osmhydrant_description">"Lookup, add and edit fire stations and fire hydrants"</string>
+    <string name="achievement_emergency_title">Life saver</string>
```

Comment:
I think it shouldn't have a space:
```suggestion
    <string name="achievement_emergency_title">Lifesaver</string>
```

But maybe that's just a British thing; this website seems particularly unsure:
https://life-saver.org.uk/

@smichel17 or other natives?

Reply:
https://en.wiktionary.org/wiki/lifesaver was also confused ( https://en.wiktionary.org/w/index.php?title=lifesaver&diff=65161919&oldid=64363252 ) - it seems that both versions are OK. Maybe one is archaic/formal?

Reply:
One person from OSM Discord recommended lifeguard for human keeping guard and lifebuoy for device.

Another "if you mean the floaty rings and stuff, 'lifesaver' is better, but both are used"

Reply:
> One person from OSM Discord recommended lifeguard for human keeping guard and lifebuoy for device.

I think they're both very specific to water/beach.

> Another "if you mean the floaty rings and stuff, 'lifesaver' is better, but both are used"

Yeah lifesaver/life saver is far more generic, a lot of the examples include paramedics, or just the general phrase "you're a lifesaver" when someone does something useful (e.g. has a spare charging cable).

Reply:
I prefer with the space.

I think @matkoniecz is on the money here— with the space is more formal and especially more serious. I agree with @peternewman that I'd probably omit the space in the phrase, "you're a lifesaver", but I don't think I'd use that phrase if someone literally saved my life. Also, "Lifesavers" reminds me of these:
<img alt="lifesaver candy roll" width="300" src="https://www.logolynx.com/images/logolynx/f0/f0108b82abf7df25d1572bd6efbe2d78.jpeg" />

I don't like lifeguard or similar because they're too water/beach specific.